package.json

```diff
@@ -86,6 +86,7 @@
         "tippy.js": "^6.3.7",
         "tslib": "^2.6.2",
         "typescript": "^5.2.2",
+        "typewriter-effect": "^2.20.1",
```

Comment:
Are you serious? This is a react library :\

Reply:
Huh! we're using its core, yea it supports react btw ( same like floatingUI ) skele skele lol
```ts
import Typewriter from "typewriter-effect/dist/core";
```

Reply:
> import Typewriter from "typewriter-effect/dist/core";

Yes no we want something better :\ . The repo is not updated in like months